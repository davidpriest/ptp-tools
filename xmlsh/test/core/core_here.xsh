# core_here.xsh
# HERE documents

echo Tabbed
xcat <<EOF
<foo>
  bar
</foo>
EOF

echo Untabbed
# Tab stripping
xcat <<-EOF
	<foo>
	bar
</foo>
EOF

exit 0