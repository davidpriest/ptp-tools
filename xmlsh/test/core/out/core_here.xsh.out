Tabbed
<foo>
  bar
</foo>
Untabbed
<foo>
bar
</foo>
